### DISCUSSION TITLE
SEB 3.0 for macOS Received Pop up - "Loading New SEB Settings Not Allowed!"

### DISCUSSION THREAD

**Discussion by 103191** [SafeExamBrowser/seb-mac#152] [Q&A]
Hi!

One of my student get the popup message Loading New SEB Settings Not Allowed! – SEB is already running in exam mode and it is not allowed to interrupt this by starting another exam. Finish the exam and quit SEB before starting another exam.

The problem is that there should not be another open exam. 

May I know how to resolve this? 
![seb-error](https://user-images.githubusercontent.com/88760782/141263256-62f6c4d3-527e-4081-883c-24448c48adaa.jpg)

SEB 3.0 for macOS
Mac OS 11. 1
Moodle 3.4.9

**Comment by danschlet** [comment]
This error message is displayed when SEB already was started using a seb(s):// link or by opening a .seb file and might have been running in the background in this case. The student should try to make sure SEB isn't running already (quit it) before opening a new configuration (in this screen shot it looks they were trying to open the downloaded config file from Safari).

We would recommend to use the much better integration with SEB in Moodle 3.9 and newer. There you don't need to send SEB config files to students and they don't need to download/open them manually.

**Comment by 103191** [comment]
Hi Danschlet,

My student said she has quite for many times, it still stated that SEB is running. Although she has quite it successfully, it still brings her to that specific exam (previous exam) when she enter the new file.

May I know is there any method to check and make sure there is no SEB running in background?

**Reply by danschlet** [reply]
She would see the SEB icon when using the app switcher (cmd - tab key shortcut).

I have an idea what the issue could be, I will try to reproduce that. You can tell her to try to reset SEB client setting (the once saved permanently which are active when SEB is started by its app icon), by using this command in the Terminal `defaults delete org.safeexambrowser.SafeExamBrowser`.

Btw., she is using the current version SEB 3.0 I guess?

**Reply by 103191** [reply]
> She would see the SEB icon when using the app switcher (cmd - tab key shortcut).
> 
> I have an idea what the issue could be, I will try to reproduce that. You can tell her to try to reset SEB client setting (the once saved permanently which are active when SEB is started by its app icon), by using this command in the Terminal `defaults delete org.safeexambrowser.SafeExamBrowser`.
> 
> Btw., she is using the current version SEB 3.0 I guess?

Hi Danschlet,

The student has tried the method you gave in above, and this help. Now, she is able to open the SEB config file successfully. 

Thank you very much.

**Comment by 103191** [comment]
Hi Danschlet,

Yes, she is using SEB 3.0.
